db/schema.rb

```diff
@@ -19,6 +19,8 @@
     t.string   "icon_url"
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
+    t.string   "rating"
```

Comment:
It got added in 494ce10f980394e62b7a26e44f24d3ec7424fd1c. Must be a mixup while working on the apps branch. Will fix.